extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfig.java

```diff
@@ -233,6 +233,36 @@ public void setPostLogoutUriParam(String postLogoutUriParam) {
         @ConfigItem(defaultValue = "false")
         public boolean splitTokens;
 
+        /**
+         * Requires that the tokens are encrypted before being stored in the cookies.
+         */
+        @ConfigItem(defaultValueDocumentation = "false")
+        public Optional<Boolean> encryptionRequired = Optional.empty();
+
+        /**
+         * Secret which will be used to encrypt the tokens.
+         * This secret must be set if the token encryption is required but no client secret is set.
+         * The length of the secret which will be used to encrypt the tokens must be 32 characters long.
```

Comment:
Does it make sense to validate the input as well as when falling back to client secret? Perhaps at build time?

Reply:
@pedroigor Sure, will add some extra check (I actually tried doing at build time - but the client secret can also be retrieved from the custom credential provider and I was not sure it would work at build time)